keg-client/src/Taps/BeerBlock.tsx

```diff
@@ -126,19 +159,47 @@ function DetailDialog(props: DetailDialogProps) {
   );
 }
 
-// function VoteButtons() {
```

Comment:
Can we move the VoteButtons component to another file? We can probably get rid of the commented out code too